src/GeneExpressionTSnePlot.js

```diff
@@ -89,7 +89,7 @@ const GeneExpressionScatterPlot = (props) => {
     }
   }
 
-  const renderGradient = plotData.max && plotData.min && plotData.max > plotData.min
+  const renderGradient = plotData.max !== null && plotData.min !== null && plotData.max > plotData.min
```

Comment:
In the case of a gene being expressed in only one cell, the values `plotData.max` and `plotData.min` will be the same. Changing `>` to `>=` will solve this.